external/derby/README.md

```diff
@@ -0,0 +1,22 @@
+# External (Third Party Container) Tests
+
+Third Party container tests help verify that the AdoptOpenJDK binaries are *good* by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/[Issue #172](https://github.com/AdoptOpenJDK/openjdk-tests/issues/172) lists the applications that we have initially targeted to best exercise the AdoptOpenJDK binaries.  For each application, we choose to run a selection of their functional tests.
```

Comment:
Change 
Third Party container tests help verify
to 
Derby tests are a part of the external third-party application tests that help verify